### ISSUE TITLE
add traceback in message if provided in logRecord

### ISSUE THREAD

**Issue by nlaurance** [AnesFoufa/python-teams-logger#12]
feature request
I wish to see the full traceback in the message body if used like 

```
    try:
        2/0
    except ZeroDivisionError as e:
        logger.error('Oops !', exc_info=True)
```

**Comment by AnesFoufa** [comment]
Thank your for this issue. I merge your pull request.

**Comment by AnesFoufa** [comment]
@nlaurance : Your PR has been published to Pypi under the tag 0.4.0
